src/lib/bootstrap.php

```diff
@@ -36,4 +36,7 @@
 $kernel->boot();
 
 // legacy handling
-require __DIR__.'/core.php';
+//require __DIR__.'/core.php';
+//ServiceUtil::setContainer($kernel->getContainer());
+//EventUtil::setManager($kernel->getContainer()->get('event_dispatcher'));
+//ModUtil::initCoreVars();
```

Comment:
these extra lines are just part of the testing I was doing. Many things work with only the set up of those three items and not the full boot up of the old core